### ISSUE TITLE
[macOS Bug Sure 20D64] WhateverGreen Kernel Panic

### ISSUE THREAD

**Issue by dreamwhite** [acidanthera/bugtracker#1458]
Hi there,
today I tried updating my external SSD macOS Big Sure Beta installation with version 11.2 RC.
I'm on a Dell Inspiron 5370 (i5-8250U + UHD 620) and I'm using OpenCore 0.6.5 STABLE + latest version of acidanthera's kexts. My EFI is available on [my personal GitHub repository](https://github.com/dreamwhite/dell-inspiron-5370-hackintosh/).

After downloading and trying to install the new OS version, after a few reboots I noticed a kernel-panic but, despite I have `-v debug=0x100 keepsyms=1` boot-args.

Anyways I managed to extract the kernel panic by booting normally my OS from my internal SSD.
It seems that the kernel-panic is caused by WhateverGreen kext which goes in backtrace.
Below the kernel-panic logs both with WhateverGreen 1.4.6 + Lilu 1.50 and with the latest builded version of them:

[kp_jan_releases.txt](https://github.com/acidanthera/bugtracker/files/5894184/kp_jan_releases.txt)
[kp_latest_commit.txt](https://github.com/acidanthera/bugtracker/files/5894186/kp_latest_commit.txt)

Do you have any clue on how to fix it? Should I try using OpenCore 0.6.6?
Moreover, may FileVault2 cause issues like that? 

Regards
dreamwhite

**Comment by dreamwhite** [comment]
# Update

It seems that Lilu crashes on `__ZN31AppleIntelFramebufferController17IsTypeCOnlySystemEv`

![IMG_20210129_150118.jpg](https://user-images.githubusercontent.com/17933708/106284303-54cd3e80-6243-11eb-83fa-df2a19f8474a.jpg)

**Comment by vit9696** [comment]
No keepsyms=1, no bug. Provide proper panic for issue reconsideration.

**Comment by dreamwhite** [comment]
> No keepsyms=1, no bug. Provide proper panic for issue reconsideration.

Ouch, I was sure that I updated the config.plist on the repository 😅
I double checked on my side and I have the proper boot-arguments:

![image](https://user-images.githubusercontent.com/17933708/106286863-6a903300-6246-11eb-84f1-5695dd575117.png)

![image](https://user-images.githubusercontent.com/17933708/106287000-96131d80-6246-11eb-97ed-20dc064711c8.png)

It doesn't make any sense that the system reboots even with the proper debugging boot-args :/ 
I'll try resetting my nvram again and boot again the system

**Comment by vit9696** [comment]
Just using Add is not enough to overwrite existing values. Read the manual carefully to understand how to update existing NVRAM values.

**Comment by dreamwhite** [comment]
> Just using Add is not enough to overwrite existing values. Read the manual carefully to understand how to update existing NVRAM values.

Of course I added the values inside Delete section. In the above screenshot I highlighted the boot-args for visual purposes.
In the Delete section I removed the boot-args:

![image](https://user-images.githubusercontent.com/17933708/106291730-47688200-624c-11eb-8475-5659ab7b5887.png)

**Comment by vit9696** [comment]
Strange, then the panic may come from before you added this I fear.

**Comment by dreamwhite** [comment]
> Strange, then the panic may come from before you added this I fear.

Mmmmh that sound strange even to me... Bob, idk, I'll test again when 11.2 will be officially released

**Comment by dreamwhite** [comment]
> > Strange, then the panic may come from before you added this I fear.
> 
> Mmmmh that sound strange even to me... Bob, idk, I'll test again when 11.2 will be officially released

Dear @vit9696 today I've updated to macOS Big Sur 20D64 (at least I tried) but I got the same message when I opened this issue.

I started questioning with my self if my configuration was wrong but it doesn't seem so. May I ask you if using a Type-C power delivery charger may interfere with the update process of macOS? I know this may sound strange but I have no idea at all 😭
